enterprise/views.py

```diff
@@ -654,12 +654,18 @@ def get_base_details(self, enterprise_uuid, course_run_id):
             raise Http404
```

Comment:
@UmanShahzad I removed the log from here but it still seems me necessary to add log here too. Actual exception is `ImproperlyConfigured` and we are raising 404 inside it.

Reply:
@asadiqbal08 The exception is raised at https://github.com/edx/edx-enterprise/blob/master/enterprise/api_client/discovery.py#L424 and the line below it. If you have an improperly configured catalog integration, this exception will constantly happen, until you configure one. Prod obviously has this configured so there's no need to log it. Moreover, the resulting stack-trace from such an error is quite unambiguous.